pkg/agent/util/net_windows.go

```diff
@@ -34,13 +34,13 @@ import (
 )
 
 const (
-	ContainerVNICPrefix  = "vEthernet"
-	HNSNetworkType       = "Transparent"
-	LocalHNSNetwork      = "antrea-hnsnetwork"
-	OVSExtensionID       = "583CC151-73EC-4A6A-8B47-578297AD7623"
-	namedPipePrefix      = `\\.\pipe\`
-	commandMaxRetry      = 5
-	commandRetryInternal = time.Second
+	ContainerVNICPrefix          = "vEthernet"
+	HNSNetworkType               = "Transparent"
+	LocalHNSNetwork              = "antrea-hnsnetwork"
+	OVSExtensionID               = "583CC151-73EC-4A6A-8B47-578297AD7623"
+	namedPipePrefix              = `\\.\pipe\`
+	enableInterfaceTimeout       = 20 * time.Second
```

Comment:
Could we add comment about how long it typically takes and any information we know why it takes so long?

Reply:
@tnqn , I'm not sure why it takes long time sometimes.

I add the log to print time to wait interface to be up. Let me run the conformance tests for several times to find the average time.

Reply:
I found I mistakenly resolved this comment and discarded one my comment earlier